signac/common/host.py

```diff
@@ -17,7 +17,7 @@
 
 
 SESSION_PASSWORD_HASH_CACHE = SimpleKeyring()
-SESSION_USERNAME_CACHE = dict()
+SESSION_USERNAME_CACHE = dict()  # type: ignore
```

Comment:
Explanation: This requests a type annotation for the `dict`. I added ignores to all deprecated modules where necessary, without questioning it.